app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,18 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(
+            self, nickname: str, favourite_dish: str, skill_level: int
+    ) -> None:
+        super(DwarfBlacksmith, self).__init__(nickname, favourite_dish)
```

Comment:
no need to pass args to `super`, since in recent python updates `super` take those args as default.
fix everywhere